README.md

```diff
@@ -121,14 +155,6 @@ and you will be provided with a web server running the site on [http://localhost
 
 Point your favorite browser to `http://localhost:1313/` and you will see the Apache Camel website.
 
-Changes that are made to the content managed by Hugo (i.e. content, layouts, config.toml) are applied automatically and reloaded in the browser. To make changes to the content managed by Antora, a rebuild needs to be done. The same is true for the CSS changes in the `antora-ui-camel`. To rebuild you can run, in another terminal window, from the root directory of the website:
```

Comment:
This should not be removed.

Reply:
I have created the changes to that and included that block as well.